docs/reference/module-options.md

```diff
@@ -372,3 +372,442 @@ module.exports = {
+
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`perPage`](#perpage) | Integer | Set the number of pieces to be |
+| [`next`](#next) | Boolean/Object | Enable and optionally configure the `req.data.next` object. |
+| [`piecesFilters`](#piecesfilters) | Array | Configure pieces filters for index pages. |
+| [`piecesModuleName`](#piecesmodulename) | String | Specify the associated piece type if it doesn't match the module name. |
+| [`previous`](#previous) | Boolean/Object | Enable and optionally configure the `req.data.previous` object. |
+
+### `perPage`
+
+For piece pages, the `perPage` option, expressed as an integer, defines the number of pieces that will be added to the `req.data.pieces` array when the page is served. The specific pieces in the set will be based on the total number of pieces and the `page` query parameter in the request.
+
+This is, more simply, the number of pieces that will normally be displayed on each page of a [paginated index page](/reference/glossary.md#index-page). This value defaults to 10.
+<!-- TODO: Change index page link to a guide on index pages when available. -->
+
+#### Example
+
+```javascript
+// modules/article-page/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    perPage: 12
+  },
+  // ...
+}
+```
+
+### `next`
+
+If set to `true`, Apostrophe will include the next piece, based on the [sort option](#sort), on `req.data.next` when serving a [show page](/reference/glossary.md#show-page). This is useful to add links to a show page directing visitors to the next item in a series (e.g., the next oldest blog post). If not set, `req.data.next` will not be available.
+
+`next` can also be set to an object, which will be used as a query builder for retrieving that next piece document.
+
+#### Example
+
+```javascript
+// modules/article-page/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    next: true
+  },
+  // ...
+}
+
+// OR
+
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    // The next article piece would be returned with only the `title`, `_url`,
+    // and `publishedAt` properties.
+    next: {
+      project: {
+        title: 1,
+        _url: 1,
+        publishedAt: 1
+      }
+    }
+  },
+  // ...
+}
+```
+
+### `piecesFilters`
+
+<!-- TODO: Link to a guide on using piece filters when available. -->
+<!-- TODO: Link to a better query builder guide when available. -->
+`piecesFilters` can be configured as an array of objects to support filtering pieces on an [index page](/reference/glossary.md#index-page). Each object must have a `name` property associated with a valid [query builder](/guide/major-changes.md#queries). These include:
+
+- Field names whose field types automatically get builders: `string`, `slug`, `boolean`, `checkboxes`, `select`, `integer`, `float`, `url`, `date`, `relationship`
```

Comment:
Should at least be a list. I'm wondering if this is the...right place for this.

Reply:
It should be in a guide about using filters for sure. I thought it'd be useful here as well.

Reply:
I think it's worth it here because even between us the question of what piecesFilters is really good for kept coming up, due to a lack of visibility for this key feature: you always get builders "for free" for most schema fields.